themes/navy/layout/partial/theme.njk

```diff
@@ -5,10 +5,15 @@
     </noscript>
     <img data-src="{{ url_for('themes/screenshots/' + plugin.name + '.png') }}" data-sizes="auto" class="plugin-screenshot-img lazyload" alt="{{ plugin.name }}">
     {% if plugin.preview %}
-    <a href="{{ plugin.preview }}" class="plugin-preview-link" target="_blank"><i class="fa fa-eye"></i></a>
+      <a href="{{ url_for('themes/screenshots/' + plugin.name + '.png') }}" class="plugin-preview-image" target="_blank"><i class="fa fa-eye"></i></a>
```

Comment:
Open a new tab instead of transferred to external link when user clicks theme thumb.

Reply:
We can use components like Fancybox or Medium Zoom to support image zooming and display.

Reply:
Yes, however, to be honest, it was a little bit tedious, so I implemented it open with new tab 😓 
I'll give it a try later.

Reply:
@stevenjoezhang

> We can use components like Fancybox or Medium Zoom to support image zooming and display.

Would you please review? @uiolee implement it in https://github.com/hexojs/site/pull/2054/commits/caec1731f475ac33e420672f93ab458e0c46d7da
It LGTM :) 

But, I can't approve this PR. Because I'm PR author...